uber/templates/macros.html

```diff
@@ -117,4 +117,81 @@
       </div>
     {%- endfor %}
   </div>
-{%- endmacro %}
\ No newline at end of file
+{%- endmacro %}
+
+{% macro address_form(group=False, attendee=False, is_prereg_dealer=False) -%}
+    {% set name_prefix='group_' if group and attendee else '' %}
```

Comment:
These first few lines are all the actual new templating logic -- we now add a `group_` prefix in certain cases. Specifically, when we call this from the groupform.html page, we try to pass in an attendee object as well -- if there is an attendee object to pass through, then that page could theoretically have attendee address fields, so we add a prefix. We then also set one variable to be either the attendee or group, prioritizing the group object.

I also moved the includes in here because scattering them all over various pages was super messy. I wanted to put the includes in base.html and control their appearance with Jinja2 variables, but ran into intractable scoping issues, so now we just make sure that this macro isn't being called on the prereg dealer page in the attendee section.